testsuite/classlibrary/TestServer_clientID_booted.sc

```diff
@@ -159,17 +160,19 @@ TestServer_clientID_booted : UnitTest {
 
 		remote1.remove;
 		thisProcess.platform.killProcessByID(serverPid);
+		exitCond.hang; // wait for the server process to exit
```

Comment:
Would we ever want this to time out so it doesn't hang indefinitely?

If so, using a mechanism like `CondVar:-waitFor` may be useful.

Reply:
Good idea! Implemented.